packages/core/theme-module/theme/pages/MyAccount/MyProfile.vue

```diff
@@ -144,14 +150,29 @@ export default {
       default: () => ({})
     }
   },
+  setup() {
+    const { user, changePassword, error } = useUser();
+    const form = ref({});
```

Comment:
I think it would be better to provide here default values for fields. for example empty strings. It's better than undefineds